tcms/telemetry/templates/telemetry/testing/breakdown.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block title %}{% trans "Testing Breakdown" %}{% endblock %}
+
+{% block contents %}
+
+    <div class="container-fluid container-cards-pf">
+
+        <form class="form-horizontal">
+            <div class="form-group">
+                <label for="id_select_product" class="col-md-1 col-lg-1">
+                    {% trans "Product" %}:
+                </label>
+                <div class="col-md-3">
+                    <select name="product" id="id_select_product" class="form-control selectpicker">
+                        <option value="">----------</option>
+                        {% for product in products %}
+                            <option value="{{ product.pk }}">
+                                {{ product.name }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                </div>
+
+                <label for="id_select_test_plan" class="col-md-1 col-lg-1">
+                    {% trans "Test Plan" %}:
+                </label>
+                <div class="col-md-3">
+                    <select name="test_plan" id="id_select_test_plan" class="form-control selectpicker">
+                        <option value="">----------</option>
+                        {% for test_plan in test_plans %}
+                            <option value="{{ test_plan.pk }}">
+                                {{ test_plan.name }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                </div>
+            </div>
+        </form>
+
+        <div class="col-md-3" style="padding: 15px; margin: 15px; text-align: center">
+            <div style="text-align: center; font-weight: bold"> {% trans "Automated/ Manual" %}</div>
+            <div id="total"></div>
+            <div> {% trans "Total" %}</div>
+
+            <div id="automated-bar" style="padding: 10px; font-weight: bold"></div>
+        </div>
+
+        <div class="col-md-4" style="padding: 15px; margin: 15px; text-align: center">
+            <div style="text-align: center; font-weight: bold">{% trans "Priorities" %}</div>
+
+            <div id="priorities-chart"></div>
+            <svg></svg>
+        </div>
+
+        <div class="col-md-4" style="padding: 15px; margin: 15px; text-align: center">
+            <div style="text-align: center; font-weight: bold">{% trans "Categories" %}</div>
+            <div id="categories-chart"></div>
```

Comment:
Bars should be stacked bars showing CONFIRMED/unconfirmed test cases.

There should also be a line chart on top of them showing the number of manual test cases. 

Try implementing them one by one after the rest is done. Depending on how it goes we may do this in a separate pull request.

Reply:
This is a lot of work, and would involve changes everywhere (API, charts). The PR is big enough now. I suggest we merge this as it is now, and this will be the next thing I will start working on